dash.el

```diff
@@ -533,7 +536,7 @@ Thus function FN should return a list."
 (defun -flatten (l)
   "Take a nested list L and return its contents as a single, flat list.
 
-Note that because `nil' represents a list of zero elements (an
+Note that because 'nil represents a list of zero elements (an
```

Comment:
maybe remove the `'` also.  Not sure which is better but since nobody quotes `nil` maybe it shouldn't be.